recipe-generator/src/main/java/ca/derekcormier/recipe/generator/filter/JavaGetterFilter.java

```diff
@@ -0,0 +1,42 @@
+package ca.derekcormier.recipe.generator.filter;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import liqp.filters.Filter;
```

Comment:
I guess you're already using this in a few places, but why not just `Function<String, Boolean>` or `interface Foo { boolean filter(String s) }` instead of using this library?

I guess this library is not part of the library (public) API so it doesn't matter? I wouldn't want the library to expose implementation details like this...

Reply:
A filter is like a function that I can call inside of the liquid templates. When I register a `liqp.filters.Filter`, I can call it by name in the template.